src/PrestaShopBundle/Resources/views/Admin/Module/Includes/card_list.html.twig

```diff
@@ -87,6 +87,7 @@
             {% if module.attributes.description|length > 0 and module.attributes.description|length < module.attributes.fullDescription|length %}
               ...
             {% endif %}
+            {{ renderhook('displayModuleCardExtraDescription', {'module': module}) }}
```

Comment:
Why hook ?
I suggest to add a new module property or method something like `getExtraDescription`

Reply:
this feature won't be native to Prestashop Open source anymore as the content comes from addons. I understand your suggestion but will it be accepted if not used in PS ?

Reply:
@sowbiba, it might end up being used by other developers, too :) from what I know, a new API allows us to connect with any API we want, not only Addons

Reply:
@PululuK suggestion sounds good 👍